Pod/Classes/Controller/PhotosViewController.swift

```diff
@@ -342,14 +342,17 @@ extension PhotosViewController {
         }
 
         // Make sure we have a data source and that we can make selections
-        guard let photosDataSource = photosDataSource, collectionView.isUserInteractionEnabled && photosDataSource.selections.count < settings.maxNumberOfSelections else { return false }
+        guard let photosDataSource = photosDataSource, collectionView.isUserInteractionEnabled else { return false }
 
         // We need a cell
         guard let cell = collectionView.cellForItem(at: indexPath) as? PhotoCell else { return false }
         let asset = photosDataSource.fetchResult.object(at: indexPath.row)
 
         // Select or deselect?
         if let index = photosDataSource.selections.index(of: asset) { // Deselect
+            if( photosDataSource.selections.count <= 0 ){
```

Comment:
Format of `if` statements you've decided to use clearly conflicts with a format (style) of other `if` statements in this project.